### PR TITLE
fix(chat): custom app completion URL validation, resolve console errors , add hint for Attachment types (Issue #2206)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

1) Completion URL Validation allow / symbol in the end 
2) Resolve console errors during custom application creation
3) Add hint for Attachment types

Issues:

- Issue #2206 

**UI changes**

Снимок экрана 2024-09-27 в 00 39 20
Снимок экрана 2024-09-27 в 00 39 50

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2211.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2211.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2211.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2211.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Common/MultipleComboBox.tsx [hunk 1]
```diff
@@ -314,46 +314,44 @@ export function MultipleComboBox<T>({
...
-                  <li className="px-3 py-2">
-                    {notFoundPlaceholder || t('No available items')}
-                  </li>
-                )}
-          </ul>
-        )}
+        <ul
+          className={classNames(
+            'z-10 max-h-80 overflow-auto rounded bg-layer-3',
+            hideSuggestions || !isOpen ? 'hidden' : '',
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Common/MultipleComboBox.tsx]:
```suggestion
            (hideSuggestions || !isOpen) && 'hidden',
```

**Review by Alexander-Kezik — DISMISSED** [review]

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**